Src/AutoFakeItEasy/FakeItEasyMethodQuery.cs

```diff
@@ -74,8 +74,46 @@ public IEnumerable<ParameterInfo> Parameters
 
             public object Invoke(IEnumerable<object> parameters)
             {
-                return new Fake<T>(
-                    b => b.WithArgumentsForConstructor(parameters));
+                var genericFakeType = typeof (Fake<>).MakeGenericType(typeof (T));

```

Comment:
I see... So, AFAICT, #693 is caused because [a public interface is now renamed in FakeItEasy 2.0.x](https://github.com/FakeItEasy/FakeItEasy/issues/520).

If that's the case, then it looks like AutoFixture.AutoFakeItEasy is not truly compatible with FakeItEasy 2.0.x. 

Instead of trying to 'hack' AutoFixture.AutoFakeItEasy, wouldn't it be easier if we apply the change in AutoFixture.AutoFakeItEasy2?